src/entities/Place/routes/getPlaceMostActiveList.ts

```diff
@@ -59,7 +59,9 @@ export const getPlaceMostActiveList = Router.memo(
       only_featured: !!bool(query.only_featured),
       only_highlighted: !!bool(query.only_highlighted),
       positions: query.positions.length
-        ? unique([...hotScenesPositions, ...query.positions])
+        ? hotScenesPositions.filter((position) =>
```

Comment:
This is a `0^2` operation, whenever you have something like this we should use a `Set` inside the filter, like:

```ts
const positions = new Set(query.positions)

hotScenesPositions.filter(position => positions.has(position))

```